src/main/java/org/prgrms/vouchermanagement/controller/api/VoucherRestController.java

```diff
@@ -0,0 +1,57 @@
+package org.prgrms.vouchermanagement.controller.api;
+
+import org.prgrms.vouchermanagement.voucher.service.VoucherService;
+import org.prgrms.vouchermanagement.voucher.voucher.Voucher;
+import org.prgrms.vouchermanagement.voucher.voucher.VoucherType;
+import org.prgrms.vouchermanagement.voucher.voucher.dto.NewVoucher;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+public class VoucherRestController {
+  /**
+   * - [*]  전체 조회기능
+   * - [*]  조건별 조회기능 (바우처 생성기간 및 특정 할인타입별)
+   * - [*]  바우처 추가기능
+   * - [*]  바우처 삭제기능
+   * - [*]  바우처 아이디로 조회 기능
+   */
+  private final VoucherService voucherService;
+
+  public VoucherRestController(VoucherService voucherService) {
+    this.voucherService = voucherService;
+  }
+
+  /**
+   * @return 모든 Voucher 목록
+   */
+  @GetMapping("/api/v1/vouchers")
+  public List<Voucher> getAllVouchers() {
+    return voucherService.getVoucherList();
+  }
+
+  @GetMapping("/api/v1/voucher/{voucherId}")
```

Comment:
REST API는 무조건 리소스를 복수로 사용합니다.
```suggestion
  @GetMapping("/api/v1/vouchers/{voucherId}")
```

Reply:
queryString으로 필터 역할로 getVoucherByVoucherId를 구현해보시겠어요?